modules/reactor/src/meshgenerators/FlexiblePatternGenerator.C

```diff
@@ -553,9 +556,25 @@ FlexiblePatternGenerator::FlexiblePatternGenerator(const InputParameters & param
 std::unique_ptr<MeshBase>
 FlexiblePatternGenerator::generate()
 {
-  if (isParamValid("external_boundary_id"))
-    MooseMesh::changeBoundaryId(
-        **_build_mesh, OUTER_SIDESET_ID, getParam<boundary_id_type>("external_boundary_id"), false);
+  const auto external_boundary_id =
+      isParamValid("external_boundary_id") ? getParam<boundary_id_type>("external_boundary_id") : 0;
+  const auto external_boundary_name =
+      isParamValid("external_boundary_name") ? getParam<std::string>("external_boundary_name") : "";
+  if (external_boundary_id > 0)
+    MooseMesh::changeBoundaryId(**_build_mesh, OUTER_SIDESET_ID, external_boundary_id, false);
+  if (!external_boundary_name.empty())
+  {
+    (*_build_mesh)
+        ->get_boundary_info()
+        .sideset_name(external_boundary_id > 0 ? external_boundary_id
```

Comment:
You can just use `external_boundary_id` here if `OUTER_SIDESET_ID` is its default value.

Reply:
OUTER_SIDESET_ID is now the default ID value